config/routes.rb

```diff
@@ -113,8 +113,8 @@
   get 'basespace/projects', to: 'basespace#projects'
   get 'basespace/samples_for_project', to: 'basespace#samples_for_project'
 
-  get '/samples/:sample_id/pipeline_viz(/:pipeline_version)', to: 'pipeline_viz#show',
-                                                              constraints: { pipeline_version: /\d+\.\d+/ } # To allow period in pipeline version parameter
+  get 'samples/:sample_id/pipeline_viz(/:pipeline_version)', to: 'pipeline_viz#show',
+                                                             constraints: { pipeline_version: /\d+\.\d+/ } # To allow period in pipeline version parameter
```

Comment:
Hm I'm pretty sure the pipeline is only ever stored as the first two numbers on `idseq-web`, since updates to the third number should indicate no changes to the results of `idseq-dag`, + we use the pipeline_version parameter here to construct part of the s3 path, which currently is only the first two numbers given the no changes to results thing above.

Not sure if this is something to guard against since it doesn't seem like it's a possibility (?)